configure.ac

```diff
@@ -424,7 +424,6 @@ m4_foreach([cflag],[
         [-Wno-c++11-narrowing],
         [-Wno-constant-logical-operand],
         [-Wno-non-pod-varargs],
-        [-Wno-overloaded-virtual],
```

Comment:
This was needed for Clang 16 not just this, so please put back thanks.

Reply:
I've compiled and run regression tests with Clang 16.0.6 and there's no more no-overloaded-virtual warnings after the 'trace' function is added. So I think it was needed just because this function was missing.